### PR TITLE
add logic if external redis is used

### PR DESCRIPTION
**Pull request by sandroded**
hi guys!
I've discovered the issue, that secret name for message broker is hardcoded in the helm chart when redis is used. I've added the logic to fix it. 
I fixed the few typos in the values.yaml: missing port in the mysql.service and name for the django debug value, and name of the key in the rabbimq secret.

### CODE REVIEW

**Comment by valentijnscholten** [comment]
Thanks. Can you base your PR against `dev`? There have been some chart changes recently there.

File: helm/defectdojo/values.yaml [hunk 1]
```diff
@@ -266,6 +266,7 @@ mysql:
   secretKey: mysql-password
   mysqlDatabase: defectdojo
   service:
+    port: 3306
```
Comment (dsever) [inline helm/defectdojo/values.yaml:269]: From next release, this port is not in use anymore. @alles-klar can give you more information

Comment (sandroded) [reply helm/defectdojo/values.yaml:269]: Shall it be removed? Because it is referenced it all deploymets at the moment as .Values.mysql.service.port

Comment (alles-klar) [reply helm/defectdojo/values.yaml:269]: after the rebase you will find the value (mysqlmaster.service.port), see https://github.com/DefectDojo/django-DefectDojo/pull/5450

**Comment by alles-klar** [comment]
Why is it a problem to have the secret name hardcoded?

File: helm/defectdojo/values.yaml (hunk 1 above)
Comment (valentijnscholten) [reply helm/defectdojo/values.yaml:269]: if the rebase is messy you can try to create a new PR  from a new branch if that's easier.

Comment (sandroded) [reply helm/defectdojo/values.yaml:269]: I'll close this one, because I created a new one: https://github.com/DefectDojo/django-DefectDojo/pull/5534

**Pull request closed without merging**